_staff/paul-shetler.md

```diff
@@ -1,15 +1,15 @@
 ---
 title: Paul Shetler
-permalink: /about/paul-shetler/
 breadcrumb: Paul Shetler
 breadcrumb_type: staff
-job-title: Chief Digital Officer
+job-title: Chief Executive Officer
 twitter-handle: paul_shetler
 banner-image: /images/staff/paul-shetler-banner.jpg
 profile-image: /images/paul-shetler-profile.jpg
-profile-excerpt: Paul Shetler - Chief Executive Officer, Digital Transformation Office.
+profile-excerpt: Paul Shetler is the CEO of the Digital Transformation Office.
```

Comment:
This will only show on his old blog posts, which are date stamped and so should reflect the info at the time of publishing.